### PR TITLE
Check in/63315/new travel logic

### PR DESCRIPTION
**Pull request by brianseek**
## Summary

This PR adds a new method for skipping travel pages based on local storage. It allows one facility per day to submit a travel claim. It is behind the travel logic feature flag.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#63315

## Testing done

Adds/updates unit tests

## What areas of the site does it impact?

Day of check-in

## Acceptance criteria

 - [ ] With the feature on: stores a history of facilityId with time stamp of submitted travel claims. Filters out travel pages if that facility has already been submitted for the day. Allows new facilities to submit travel on the same day.
 - [ ] With the feature off: limits travel only to appointment payloads that have no more than one appointment.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/check-in/utils/navigation/navigation.unit.spec.js [hunk 1]
```diff
@@ -238,6 +238,91 @@ describe('Global check in', () => {
...
+        ];
+        expect(appointments).to.have.lengthOf(2);
+        const form = updateFormPages(
+          patientDemographicsStatus,
+          testPages,
+          URLS,
+          isTravelReimbursementEnabled,
+          appointments,
+          true,
+          {},
```
Comment (wullaski) [inline src/applications/check-in/utils/navigation/navigation.unit.spec.js:319]: Maybe this test or add another where there is a claim but with a different facility.

**Review by wullaski — CHANGES_REQUESTED** [review]
Looks good just one missing test case maybe.

**Review by lee-delarm6 — APPROVED** [review]
Looks good after the test addition John suggested

**Review by wullaski — APPROVED** [review]
LGTM!

**Pull request merged**